cuml/umap_demo.ipynb

```diff
@@ -34,36 +26,62 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
     "import numpy as np\n",
-    "import pandas as pd\n",
     "\n",
-    "import cudf\n",
-    "import os\n",
+    "import pandas as pd\n",
+    "import cudf as gd\n",
     "\n",
     "from sklearn import datasets\n",
+    "\n",
     "from sklearn.metrics import adjusted_rand_score\n",
     "from sklearn.cluster import KMeans\n",
+    "\n",
     "from sklearn.manifold.t_sne import trustworthiness\n",
     "\n",
-    "from cuml.manifold.umap import UMAP"
+    "from umap import UMAP\n",
+    "from cuml.manifold.umap import cumlUMAP"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Running cuml's UMAP model on blobs dataset"
+    "## Generate Blobs Data"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# create a blobs dataset with 500 samples and 10 features each\n",
-    "data, labels = datasets.make_blobs(\n",
-    "    n_samples=500, n_features=10, centers=5)"
+    "n_samples = 500\n",
+    "n_features = 10\n",
+    "n_centers = 5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'datasets' is not defined",
```

Comment:
I think the imports cell was not run?

Reply:
The notebooks should be getting cleared before getting checked in. I might have accidentally executed a umap cell and checked it in. I’ll clean them all before this is merged.